memcache/memcache.go

```diff
@@ -65,7 +65,7 @@ var (
 
 const (
 	// DefaultTimeout is the default socket read/write timeout.
-	DefaultTimeout = 100 * time.Millisecond
+	DefaultTimeout = 500 * time.Millisecond
```

Comment:
Has the current default timeout for requests ever bitten us?

Reply:
I don't think so. Let's try with this and see what happens.